modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase

```diff
@@ -1167,6 +1167,58 @@ definition {
 		AppBuilderAdmin.validateItemNameOnTable(itemName = "Content of form deployed as widget.");
 	}
 
+	@description = "Search for a deployment in the deployment lists"
+	@priority = "5"
+	test SearchForDeployedItem {
+		property portal.acceptance = "false";
+
+		AppBuilderAdmin.addCustomObject(
+			customObjectName = "my custom object",
+			isCreatingFormView = "true",
+			isCreatingViaPlusIcon = "true");
+
+		AppBuilderAdmin.createFormViewAndTableView(
+			fieldType = "Text Field",
+			formViewName = "My Form View",
+			tableViewName = "My Table View");
+
+		App.add(
+			appTitle = "First App",
+			deployOption = "Product Menu",
+			formViewName = "My Form View",
+			tableViewName = "My Table View");
+
+		App.add(
+			appTitle = "Second App",
+			deployOption = "Product Menu",
+			formViewName = "My Form View",
+			tableViewName = "My Table View");
+
+		App.add(
+			appTitle = "Third App",
+			deployOption = "Product Menu",
+			formViewName = "My Form View",
+			tableViewName = "My Table View");
+
```

Comment:
Hi @nkkwon , after adding the 3 apps, I would suggest you:

Validation 4 (make sure the list of deployments are ordered by last modified date by default - most recente on top - you can look if the first item is Third App in the table, and First App is the last one in the table)

Then you perform the search by the term: "App" so you can cover the following validations: 
Validation 1 (search and retrieve **all** deployments that has a specific "term" - so we'll be testing "contains" search ).  
Validation 3 (search and see if the items are ordered by last modified date)
Additionally you can search by a unique term "First" as you were doing, so you can guarantee that the exact match search is also working.

Then, search by an invalid term:
Validation 5.

Then, clear the search results to do the following validations:
Validation 6 (make sure that all the deployments are shown again

Then, finish with Validation 2 (order the list of deployments by Name/Title;

Reply:
Added the changes!